README.md

```diff
@@ -177,12 +177,20 @@ The client specified MUST implement functions called `getObject` and `putObject`
 
 Sets the `Cache-Control` header on the uploaded files.
 
+This can also be a function. The function will be called for each file, with the
```

Comment:
@achambers Because I feel bad for duplicating the whole S3 configuration, across 3 deploy targets, just to change a header on 1 file.

When testing this I ended up returning a function from the config function.